I1724 ibm style guide (#1221)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_writing-a-devfile-for-your-project.adoc

[id="specifying-multiple-projects-in-a-devfile_{context}"]
== Specifying multiple projects in a devfile

> A single devfile can specify multiple projects. For each project, specify the type of the source repository, its location, and, optionally, the directory the project will be cloned to.

Reviewer: ```suggestion
A single devfile can specify multiple projects. For each project, specify the type of the source repository, its location, and, optionally, the directory the project is cloned to.
```
